tests/installation/mediacheck.pm

```diff
@@ -27,12 +28,18 @@ sub run {
     ensure_shim_import;
     $self->select_bootmenu_more('inst-onmediacheck', 1);
 
-    # the timeout is insane - but some old DVDs took almost forever, could
-    # recheck with all current one and lower again
-    assert_screen [qw(mediacheck-ok mediacheck-checksum-wrong)], 3600;
-    send_key "ret";
-    if (match_has_tag('mediacheck-checksum-wrong')) {
-        die "Checksum reported as wrong";
+    while ($iterations++ < 3) {
```

Comment:
Is it kind of workaround? What is the reason to have a loop here?

Reply:
Thank you @rwx788; this patch is not a workaround, the loop is present to continue to support the "old" linuxrc version for release other than TW that do not have yet support of this new screen.